web/webpack.config.ts

```diff
@@ -146,6 +146,7 @@ export default (
                             "rendered_markdown",
                             "tooltip_hotkey_hints",
                         ],
```

Comment:
Should we also be setting `knownHelpersOnly` (https://handlebarsjs.com/api-reference/compilation.html#handlebars-compile-template-options)? I don't think we have any dynamic helper generation.

I suppose `explicitPartialContext` might also be a good refactor, but definitely out of scope for this PR.